### PR TITLE
Add support for reconnect window with slates

### PR DESCRIPTION
**Pull request by jaredsmith**
Also change one test due to https://docs.mux.com/changelog/mux-data-video-views-api-id-format-changed-to-match-the-video-id?month=jul&year=2022

Signed-off-by: Jared Smith <jsmith@mux.com>

### CODE REVIEW

File: examples/data/exercise-video-views.rb [hunk 1]
```diff
@@ -24,7 +25,8 @@
 
 # ========== get-video-view ==========
 view = views_api.get_video_view(video_views.data.first.id)
+puts "JSMITH: " + view.data.id
```
Comment (jsanford8) [inline examples/data/exercise-video-views.rb:28]: I think you want to remove this

**Review by jsanford8 — CHANGES_REQUESTED** [review]
This all looks pretty good to me, just one comment about some debug strings and then also let's move any double-space-after-period to a single space to be consistent with the rest. I only commented on a few of those occurrences, but a global find/replace would be pretty easy

File: examples/data/exercise-video-views.rb (hunk 1 above)
Comment (jaredsmith) [reply examples/data/exercise-video-views.rb]: I did!

File: docs/Asset.md [hunk 2]
```diff
@@ -25,7 +25,7 @@
 | **source_asset_id** | **String** | Asset Identifier of the video used as the source for creating the clip. | [optional] |
 | **normalize_audio** | **Boolean** | Normalize the audio track loudness level. This parameter is only applicable to on-demand (not live) assets. | [optional][default to false] |
 | **static_renditions** | [**AssetStaticRenditions**](AssetStaticRenditions.md) |  | [optional] |
-| **recording_times** | [**Array&lt;AssetRecordingTimes&gt;**](AssetRecordingTimes.md) | An array of individual live stream recording sessions. A recording session is created on each encoder connection during the live stream | [optional] |
+| **recording_times** | [**Array&lt;AssetRecordingTimes&gt;**](AssetRecordingTimes.md) | An array of individual live stream recording sessions. A recording session is created on each encoder connection during the live stream.  Additionally any time slate media is inserted during brief interruptions in the live stream media or times when the live streaming software disconnects, a recording session representing the slate media will be added with a \&quot;slate\&quot; type. | [optional] |
```
[the remark this answers is not in the document]
Comment (jaredsmith) [reply docs/Asset.md]: I updated this in the openapi-specification, and then rebuild things here.

File: docs/CreateLiveStreamRequest.md [hunk 3]
```diff
@@ -6,14 +6,16 @@
...
+| **reconnect_window** | **Float** | When live streaming software disconnects from Mux, either intentionally or due to a drop in the network, the Reconnect Window is the time in seconds that Mux should wait for the streaming software to reconnect before considering the live stream finished and completing the recorded asset. Defaults to 60 seconds on the API if not specified.  Reduced and Low Latency streams with a Reconnect Window greater than zero will insert slate media into the recorded asset while waiting for the streaming software to reconnect or when there are brief interruptions in the live stream media. When using a Reconnect Window setting higher than 60 seconds with a Standard Latency stream, we highly recommend enabling slate with the &#x60;use_slate_for_standard_latency&#x60; option.  | [optional][default to 60] |
```
[the remark this answers is not in the document]
Comment (jaredsmith) [reply docs/CreateLiveStreamRequest.md:9]: Here as well.

File: docs/CreateLiveStreamRequest.md [hunk 4]
```diff
@@ -6,14 +6,16 @@
...
+| **use_slate_for_standard_latency** | **Boolean** | By default, Standard Latency live streams do not have slate media inserted while waiting for live streaming software to reconnect to Mux.  Setting this to true enables slate insertion on a Standard Latency stream. | [optional][default to false] |
+| **reconnect_slate_url** | **String** | The URL of the image file that Mux should download and use as slate media during interruptions of the live stream media.  This file will be downloaded each time a new recorded asset is created from the live stream.  If this is not set, the default slate media will be used. | [optional] |
```
[the remark this answers is not in the document]
Comment (jaredsmith) [reply docs/CreateLiveStreamRequest.md]: And here as well

File: docs/CreateLiveStreamRequest.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (jaredsmith) [reply docs/CreateLiveStreamRequest.md:9]: This came from the OpenAPI spec, so I've fixed it there, and then re-build this based on the new spec.

File: examples/data/exercise-video-views.rb (hunk 1 above)
Comment (jaredsmith) [reply examples/data/exercise-video-views.rb]: Fixed

**Review by jsanford8 — APPROVED** [review]

**Pull request merged**